CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ Here are the basic steps to get started with your first contribution. Please rea
 5. Install development requirements. `pip install -r dev-requirements.txt`
 6. Create a test that replicates the issue.
 7. Make code changes.
-8. Ensure unit tests pass and code style / formatting is consistent (see [wiki](https://github.com/Microsoft/Recommenders/wiki/Coding-Guidelines#python-and-docstrings-style) for more details).
+8. Ensure that unit tests pass and code style / formatting is consistent (see the [coding guidelines](https://github.com/Microsoft/Recommenders/wiki/Coding-Guidelines#python-and-docstrings-style) for more details). In particular, make sure that there is a docstring for every function and class you add and that it conforms to the Google style. 
```

Comment:
maybe it would be good to add a link to Google docs:

```
In particular, make sure that there is a docstring for every function and class you add and that it conforms to the [Google style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).
```